roboteam_ai/src/Switches.cpp

```diff
@@ -297,14 +301,18 @@ bt::Node::Ptr Switches::tacticSwitch(std::string name, bt::Blackboard::Ptr prope
 
     std::map<std::string, std::vector<std::pair<std::string, robotType>>> tactics = {
 
-            // Keeper tactics
+            // Keeper tactics. The rolename Keeper is hardcoded in our system, so use it!
```

Comment:
oh no... this doesn't sound too good

Reply:
Okay after further reviewing I did not see the string 'keeper' anywhere, so i presume that there has been no behavior change or hardcoded dependency on the rolenome keeper. I presume you mean that the name 'keeper' needs to be that because it also needs to be that in trees? (like with all our roles)